internal/cloudformation/tasks/upsert.go

```diff
@@ -88,11 +88,34 @@ func upsertStack(
 	checkError(err)
 
 	if *changeSet.Status == "FAILED" {
-		printer.Fatal(
-			fmt.Errorf("Cloudformation ChangeSet failed to create"),
-			*changeSet.StatusReason,
-			"",
-		)
+		if *changeSet.StatusReason == "The submitted information didn't contain changes. Submit different information to create a change set." {
+			printer.Error(
+				fmt.Errorf("Cloudformation ChangeSet failed to create: no changes"),
+				*changeSet.StatusReason,
+				"",
+			)
+			printer.SubStep("Cleaning up unused change set", 1, true, true)
+			_, err := cf.DeleteChangeSet(
```

Comment:
This is causing a test failure, since the cfn wrapper is called `client` rather than `cf`.